styles/_variables.scss

```diff
@@ -9,15 +9,15 @@ $secondary: adjust-color($primary, $lightness: -4%);
 $background: adjust-color($primary, $lightness: -10%);
 $border: adjust-color($primary, $lightness: -14%);
 $border2: adjust-color($primary, $lightness: 100%);
-$focusedBorder: rgb(0, 0, 0);
+$focusedBorder: adjust-color($text, $lightness: 35%);
 $emphasis1: rgb(253,166,57);
 $emphasis2: rgb(174, 138, 88);
 $error: red;
-$overlay: adjust-color($primary, $lightness: -56%, $alpha: -0.35);
+$overlay: adjust-color($secondary, $lightness: 0%, $alpha: -0.1);
```

Comment:
$lightness: 0% doesn't do anything, you can write this as adjust-color($secondary, $alpha: -0.1).